web/locale/en_US/messages.po

```diff
@@ -2696,17 +2696,17 @@ msgstr "No invites found"
 msgid "How do I choose?"
 msgstr "How do I choose?"
 
-msgid "Champion - A CoderDojo Champion is an individual who spearheads setting up and maintaining a Dojo. Champions do not necessarily have to have the ability to code but possess the skills required to bring together technical mentors and supporters to run a Dojo frequently!"
-msgstr "Champion - A CoderDojo Champion is an individual who spearheads setting up and maintaining a Dojo. Champions do not necessarily have to have the ability to code but possess the skills required to bring together technical mentors and supporters to run a Dojo frequently!"
+msgid "Champion - CoderDojo Champions are those who take the lead in organising a Dojo and ensure each session runs smoothly. Champions can be one person or it can be a few people who Co-Champion a Dojo to share the work. Champions should join their Dojo on Zen to set up events, award badges and to contribute to the CoderDojo Forums."
```

Comment:
thought abt that, need confirmation from @rosalanghammer as i just copy-pasted the content from her description :D